README.md

```diff
@@ -18,210 +18,294 @@ Darkflame Universe is licensed under AGPLv3, please read [LICENSE](LICENSE). Som
+sudo apt-add-repository 'deb https://apt.kitware.com/ubuntu/ $(lsb_release -cs) main'
 
-#### Build the repository
+# Next you'll want to ensure that Kitware's keyring stays up to date
+sudo apt update
+sudo apt install kitware-archive-keyring
+sudo rm /etc/apt/trusted.gpg.d/kitware.gpg
 
-You can either run `build.sh` when in the root folder of the repository:
+# If sudo apt update above returned an error, copy the public key at the end of the error message and run the following command
+# if the error message was "The following signatures couldn't be verified because the public key is not available: NO_PUBKEY 6AF7F09730B3F0A4"
+# then the below command would be "sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 6AF7F09730B3F0A4"
+sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys <TheCopiedPublicKey>
 
-```bash
-./build.sh
+# Finally update and install
+sudo apt update
+sudo apt install cmake
 ```
 
-Or manually run the commands used in `build.sh`:
+#### Verifying your CMake install
+Verify that your `CMake` version is at least 3.18 or higher!  You can see the command to figure that out [here](#cmake)
+
+### Packages
+
+#### Linux build packages
+Make sure packages like `gcc`, and `zlib` are installed. Depending on the distribution, these packages might already be installed. Note that on systems like Ubuntu, you will need the `zlib1g-dev` package so that the header files are available. `libssl-dev` will also be required as well as `openssl`.  You will also need a MySQL database solution to use.  We recommend using `mariadb-server`.
+For Ubuntu, you would run the following commands.  On other systems, the package install command will differ.
 
 ```bash
-# Create the build directory, preserving it if it already exists
-mkdir -p build
-cd build
+sudo apt update && sudo apt upgrade
 
-# Run CMake to generate make files
-cmake ..
+sudo apt install build-essential
 
-# To build utilizing multiple cores, append `-j` and the amount of cores to utilize, for example `cmake --build . --config Release -j8'
-cmake --build . --config Release
-```
+sudo apt install gcc
 
-### MacOS builds
-Ensure `cmake`, `zlib` and `open ssl` are installed as well as a compiler (e.g `clang` or `gcc`).
+sudo apt install zlib1g-dev
 
-In the repository root folder run the following. Ensure -DOPENSSL_ROOT_DIR=/path/to/openssl points to your openssl install location
-```bash
-# Create the build directory, preserving it if it already exists
-mkdir -p build
-cd build
+sudo apt install libssl-dev
 
-# Run CMake to generate build files
-cmake .. -DOPENSSL_ROOT_DIR=/path/to/openssl
+sudo apt install openssl
 
-# Get cmake to build the project. If make files are being used then using make and appending `-j` and the amount of cores to utilize may be preferable, for example `make -j8`
-cmake --build . --config Release
+sudo apt install mariadb-server
+
+# On some systems, this package does not exist.
+sudo apt install zlib
```

Comment:
You can turn this into a single apt command, which will make this significantly more condensed and not really lose you anything